test/entrypoint.sh

```diff
@@ -22,5 +88,13 @@ check_return_code
 # Deprecated Image
 curl -X 'GET' 'https://catalog.redhat.com/api/containers/v1/repositories/registry/registry.access.redhat.com/repository/rhscl%2Fnodejs-8-rhel7' -H 'accept: application/json' > image.json
 
+# Test parse_hacbs_test_output
+test_parse_hacbs_test_output(){
```

Comment:
I think we'll also need to call this function within this file in order to execute it in CI.

Reply:
Yes, thanks.